### ISSUE TITLE
lang reset to locale every reload

### ISSUE THREAD

**Issue by lamaresh** [SchizoDuckie/DuckieTV#433]
As you can see in the screenshot selected language is english, but interface language is italian, my locale. This happens every time i open DuckieTV. Same bug of this: https://github.com/SchizoDuckie/DuckieTV/issues/285

![lang](https://cloud.githubusercontent.com/assets/3250975/7672574/519b2536-fcf7-11e4-8c3a-ac586a82ba69.jpg)

**Comment by SchizoDuckie** [comment]
Thanks for reporting! we will investigate.

**Comment by garfield69** [comment]
Research in my windows sandbox revealed that only en_us has this problem.
Setting the Chrome local language to any non-English setting highlighted this for me. 
Digging deeper, I discovered that by removing `.fallbackLanguage(['en_us'])`  from the app.translate.js fixed the issue.
I've checked the angular-translate docs and we are using the fall-back correctly, even tried `.fallbackLanguage('en_us')` but this made no difference.
[Issue 764 on angular-translate](https://github.com/angular-translate/angular-translate/issues/764) is possibly a close match to our problem, its not clear if there will be a solution forthcoming. 
So my simple solution is to do without the fall-back altogether, and make all the translation files have a complete set of keys. This is not a big deal as for all files (except en_uk)  have only the `SETTINGS/DISPLAY/locale/*` keys missing.
